sdk/communication/communication-administration/package.json

```
@@ -20,6 +20,8 @@
     "check-format": "prettier --list-different \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "clean": "rimraf dist dist-* types *.tgz *.log",
     "execute:samples": "npm run build:samples && dev-tool samples run dist-samples/javascript dist-samples/typescript/dist/dist-samples/typescript/src/",
```

Is there a reason why this is not with two dev-tools invocations?
```suggestion
    "execute:samples": "npm run build:samples && execute:js-samples && execute:js-samples",
```